docker/observability/promtail/config.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://loki:3100/loki/api/v1/push
+
+scrape_configs:
+  # default config from the container, scraping itself
+  - job_name: system
+    static_configs:
+    - targets:
+        - localhost
+      labels:
+        job: varlogs
+        __path__: /var/log/*log
+  # docker service discovery
+  - job_name: docker
+    docker_sd_configs:
+      - host: unix:///var/run/docker.sock
+        refresh_interval: 5s
+        # filters:
```

Comment:
Is this commented out on purpose here?

Reply:
yeah, but I should just remove it instead of leaving it commented. A Promtail deep-dive is out of scope for this ticket :sweat_smile: